src/functional.py

```diff
@@ -1,7 +1,5 @@
 import nipype.interfaces.io as nio           # Data i/o 
 import nipype.interfaces.spm as spm          # spm
-import nipype.interfaces.matlab as mlab      # how to run matlab
-import nipype.interfaces.fsl as fsl          # fsl
 import nipype.interfaces.freesurfer as fs    # freesurfer
 import nipype.pipeline.node_wrapper as nw    # nodes for pypelines
```

Comment:
This is a better way
```suggestion
import nipype.pipeline.node_wrapper as nwww   # nodes for pypelines
```